### PR TITLE
Add initial genesis

### PR DESCRIPTION
**Pull request by albertandrejev**

### CODE REVIEW

**Review by foxpy — COMMENTED** [review]
Looks alright for me (I guess? I don't have an idea what a good genesis looks like).

Though I wish there was a little documentation about accounts. I see there are five accounts (with 3 of them used for validators), I guess one of two remaining ones is for faucet and an additional one is for relayer?

**Comment by albertandrejev** [comment]
> ...an additional one is for relayer?

No, it contains HUGE amount of coins.

**Comment by albertandrejev** [comment]
btw @foxpy you can check different gov params, minting params etc.

**Comment by foxpy** [comment]
> No, it contains HUGE amount of coins.

Alright, I see, it is defined on line 112.

**Comment by albertandrejev** [comment]
> Alright, I see, it is defined on line 112.

No, it is faucet

**Comment by foxpy** [comment]
> No, it is faucet

:|

alright, we really need a doc for accounts in this genesis.

**Comment by albertandrejev** [comment]
> we really need a doc for accounts in this genesis.

Got it

**Comment by albertandrejev** [comment]
> we really need a doc for accounts in this genesis.
Ready https://www.notion.so/Neutron-a1da12ab92fe4afbaae39d1b5c79c5c5#0fb4df24648a4bbfb0915f14c0c10a91

**Comment by foxpy** [comment]
Alright, thank you. So we don't have any genesis account(s) for relayers. Maybe we should create it/them?

**Comment by foxpy** [comment]
> So we don't have any genesis account(s) for relayers. Maybe we should create it/them?

CC @albertandrejev

**Comment by albertandrejev** [comment]
I think we can create it without recreating genesis. We can get required amount of funds from "huge account".

**Review by foxpy — APPROVED** [review]
> We can get required amount of funds from "huge account".

Doesn't sound like the fanciest solution, but I am okay with it.

**Comment by albertandrejev** [comment]
> Doesn't sound like the fanciest solution, but I am okay with it.

Also we can use `feegrant` module and `fee_granter` option for the hermes configuration, in this case we will not be required to top up dedicated address for relayer.

**Comment by sotnikov-s** [comment]
> > we really need a doc for accounts in this genesis.
> 
> Ready https://www.notion.so/Neutron-a1da12ab92fe4afbaae39d1b5c79c5c5#0fb4df24648a4bbfb0915f14c0c10a91

wdyt about adding more info about the genesis file to the readme? like this:

neutron-devnet-1/genesis.json is a genesis file for neutron devnet, blah-blah, the addresses are by the link: the-above-link, some other details

**Comment by albertandrejev** [comment]
> wdyt about adding more info about the genesis file to the readme? like this:

Done, please take a look.

**Comment by NeverHappened** [comment]
How about we rename project to `neutron-assets`?

**Comment by albertandrejev** [comment]
> How about we rename project to `neutron-assets`?

tbh from my point of view it looks redundant. We already have `neutron-org`. But if you want to be consistent then we can do it. But in this case all organization's repos names will look at least strange. All repos name will start with `neutron-*`

File: neutron-devnet-1/genesis.json [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+          "controller_enabled": true
+        }
+      },
+      "host_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "port": "icahost",
+        "params": {
+          "host_enabled": true,
+          "allow_messages": []
```
[the remark this answers is not in the document]
Comment (NeverHappened) [reply neutron-devnet-1/genesis.json:427]: Also look into other variables changes on init.sh - maybe something is important besides allowed_messages

[the remark this answers is not in the document]
Comment (albertandrejev) [reply neutron-devnet-1/genesis.json]: Implemented

**Review by NeverHappened — APPROVED** [review]
LGTM now

**Pull request merged**